.github/workflows/publish-snapshot.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish snapshot
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - ray/ui-update
+
+jobs:
+  publish-snapshot :
+    runs-on : ubuntu-latest
+    if : github.repository == 'square/workflow-kotlin'
+    timeout-minutes : 25
+
+    steps :
+      - uses: actions/checkout@v2
+      - uses: gradle/wrapper-validation-action@v1
+      - name: set up JDK 11.0.7
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11.0.7
+
+      - name : Publish Snapshots
+        run: ./gradlew clean build && ./gradlew publish --no-parallel --no-daemon
```

Comment:
Do we need `--no-daemon` on both sides?

```
        run: ./gradlew clean build --no-daemon && ./gradlew publish --no-parallel --no-daemon
```

Reply:
Yep.  They're totally different commands thanks to `&&`.

Reply:
☝️  🤦   I was on my phone and read that backwards.  It's not a huge deal but yes I'll make an update.